client/landing/stepper/declarative-flow/test/site-migration-flow.tsx

```diff
@@ -1,19 +1,24 @@
 /**
  * @jest-environment jsdom
  */
+import { isCurrentUserLoggedIn } from '@automattic/data-stores/src/user/selectors';
```

Comment:
It is necessary to enable the test suite to pass the login requirement redirect.